### ISSUE TITLE
Bingqing Wang falun gong suicide

### ISSUE THREAD

**Issue by cirosantilli** [cirosantilli/chat#5]
> You think so because you don't know Falun Gong. A friend of my family suicide by jumping from 6th floor because of Falun Gong.

bingqingwang https://twitter.com/bingqingwang/status/597900825789214721

**Comment by askej** [comment]
Sure, I don't like discussing politics but just want to share the story.  This woman was the wife of my mother's coworker.  She practiced Falun Gong for like a year and started complaining to her husband about a ghost in the apartment chasing her.  Sometimes she was so frightened and wanted to run away from her apartment.  Later she jumped out from a window.

This happened when I was a kid - 1997 or 98.  I am very sure this happened before the government's ban on Falun Gong (which was in 1999), so this has nothing to do with the government.  

Sure you can argue the suicide is not because of Falun Gong, maybe some psychological disorder.  But anyway Falun Gong did not save her.

**Comment by askej** [comment]
BTW I am not sure what she does for work but I believe she was highly educated (her husband was a professor in engineering school).  I was shocked that a professor's wife believed in ghosts.

**Comment by cirosantilli** [comment]
@askej thanks for sharing this sad story

From what you've said, there I see no clear relation between the fact that she did Falun Gong and her killing herself, is it really fair to attribute it to Falun Gong?

I would guess that she started practising FLG after she felt the symptoms of a disorder (depression?). Then, supposing she did only that instead of going modern treatment, we get back to the No1 anti FLG argument: http://skeptics.stackexchange.com/questions/27529/have-many-falun-gong-practitioners-forgone-medical-treatment-and-died-of-treatab

Of course, those questions are hard to evaluate... I've known a reasonably close depression suicide case, and no modern treatment could save her :'-(

What we'd need are (FLG x suicide) statistics to decide I suppose.

Peace.

**Comment by askej** [comment]
Right, it is hard to say, probably unfair to attribute everything to FLG.  I just talked with my mom about this and she confirmed that the incident happened before the ban.  I was wrong with "practicing FLG for a year".  She was longer than that, starting from tranditional Qi Gong, then other variants of Qi Gong, and later Falun Gong.  Among them FLG was the most "brainwashing" version.  Her husband blamed her death to all these practices mixed up.

She is the only FLG follower that I know before the ban.  Also she is the only person who committed suicide around my circle.  It may be unfair but it is hard for me not connecting these two.  

I emphasized the time because everything changed since the government ban.  After the ban, FLG became more like a political group rather than religion.  As far as I know, it became less aggressive in terms of practicing, but more active engaging with the government.  Many people joined FLG not because they believe in FLG, but because it is a much easier way to gain US green card - it takes 5-7 years for a Chinese engineer to acquire US green card via EB2 visa, but just 1 year if one claims joining FLG and applies refugee green card.  FLG followers and FLG itself nowadays are totally different from 20 years ago.  FLG appears no harm today, but I feel cold when looking at the past of FLG.

**Comment by cirosantilli** [comment]
Thanks for taking the time to investigate the matter! What you say makes sense.

I didn't know about the USA Visa angle, it is an interesting point. My GF and the few practitioners I've seen seem to really believe in it, but of course I don't know about other places. It is a shame that there are dishonest people who take advantage of it as well.

FLG is definitely a political force, like other groups of people who have a similar view on how to live. Likely the ban exacerbates the political side: if someone says: "hey, it is now illegal to do that thing which you love" people's reaction will be to protest and fight against the government. But I think their force comes from people actually believing in it, otherwise why go to so much trouble.
